src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+import duke.command.DukeCommands;
+import duke.util.*;
```

Comment:
Should more specific imports be used here instead?

Reply:
addressed in ca26cb67916444729632edba7164a8834ff5f6e2